docs/organization-integration/upload-instructions.md

```diff
@@ -81,16 +81,32 @@ If you are uploading from a Unix, Linux or macOS system, then you probably are i
 
 If you are uploading from Windows, you likely need to create a new key. You can, for example, use the [puttygen3](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html) application to accomplish this.
 
-### Uploading to portal.sig.eu via scp
+### Uploading to upload.sigrid-says.com via scp
 
 Connections to our upload server can be made using an SCP client, such as [WinSCP](http://winscp.net/eng/index.php) for Windows, or the command line utility `scp` for Unix, Linux and macOS, which is part of the [OpenSSH](http://www.openssh.com) suite.
 
-Below an example for the secure copy command, which refers to a private key, the zip file to be uploaded and 'your-upload-account' that you will receive from Sigrid support.
+An example of the secure copy command scp, which refers to a private key, the zip file to be uploaded and 'your-upload-account' that you will receive from Sigrid support. If you’re leaving out the remote file name, the ‘:’ at the end of ‘upload.sigrid-says.com’ is essential. The warning scp: remote fsetstat: Operation unsupported is harmless.
```

Comment:
Perhaps put 'scp: remote fsetstat: Operation unsupported' between quotes for readability